channels/us.m3u

```diff
@@ -2536,7 +2536,7 @@ https://stitcheraws.unreel.me/wse-node02.powr.com/live/5bf2218bd5eeee0f5a40941b/
 https://dai2-xumo.fubo.tv/amagi_hls_data_xumo1212A-motorvisiontv/CDN/master.m3u8
 #EXTINF:-1 tvg-id="MovieKingdom.us" tvg-name="Movie Kingdom" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pIuJqYk.png" group-title="Movies",Movie Kingdom
 https://a.jsrdn.com/broadcast/e9b4093a41/+0000/high/c.m3u8
-#EXTINF:-1 tvg-id="MovieMex.us" tvg-name="Movie Mex" tvg-country="US" tvg-language="Spanish" tvg-logo="" group-title="Movies",Movie Mex
+#EXTINF:-1 tvg-id="Moovimex.us" tvg-name="Moovimex" tvg-country="US" tvg-language="Spanish" tvg-logo="" group-title="Entertainment",Moovimex
```

Comment:
Shouldn't it remain in Movies category and tvg-country be MX (Mexico?)

Reply:
Well, according to the sites I read, they do narco series and other programming. I can't find a logo anywhere for it. That's how I learned about the other shows.

Moovimex: Moovimex is a collection of Mexican action, narco and comedy movies.
Pongalo NovelaClub: Pongalo NovelaClub offers hit telenovelas and TV series from throughout Latin America all in Spanish.

Reply:
No need to revert anything if you can prove you're right. You can make tvg-country MX tho

Reply:
I'm not sure on the country, I can change it to what you guys think is best. As you can see, I didn't edit that part.